pyscf/md/test/test_velocverlet.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+# Copyright 2014-2018 The PySCF Developers. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import numpy as np
+from pyscf import gto, scf
+import pyscf.md as md
+
+CHECK_STABILITY = False
+
+h2o = gto.M(verbose=3,
```

Comment:
Initialization can be put in the `setUpModule` function. The default test framework is pytest now. With pytest the module level initialization may hit resources limits in the github CI pipelines.

Reply:
Pushed the setup into the `setUpModule` function.